packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -9,11 +9,40 @@ import Inject from '../../Inject';
 import getDefaultT from '../../translate';
 import TooltipTrigger from '../../TooltipTrigger';
 
-function TitleSubHeader({ title, iconId, loading, inProgress, editable, getComponent, ...rest }) {
+function TitleSubHeader({
+	title,
+	iconId,
+	loading,
+	inProgress,
+	editable,
+	getComponent,
+	onEdit,
+	onCancel,
+	onSubmit,
+	...rest
+}) {
+	const [isEditMode, setEditMode] = React.useState(false);
```

Comment:
I’d also change the `setEditMode` to `setIsEditMode`, as that seems to be the convention: https://reactjs.org/docs/hooks-custom.html
(“setting the variable ‘isEditMode’” as opposed to “setting the editMode”)

But I don't know if it's really important and if we follow it or not, I approve either way